hack/gl-search.sh

```diff
@@ -46,7 +46,7 @@ if [ "$gls_gl_dist" != "today" ]; then
   fi
 fi  
 
-repo_url="$base_url/dists/${gls_gl_dist}/Release?ignoreCaching=1"
+repo_url="$base_url/dists/${gls_gl_dist}/InRelease?ignoreCaching=1"
```

Comment:
we can probably get rid of the `ignoreCaching` pseudo query part, as the new repo pipeline explicitly invalidates all caches when needed (https://github.com/gardenlinux/repo/blob/main/.github/workflows/build.yml#L129)